### ISSUE TITLE
[Feature] Bots will gather around a message beacon.

### ISSUE THREAD

**Issue by warestarstuff** [randomperson189/ZSBots#3]
Placing down a message beacon with the message "meet up here" will make the bots attempt to gather around that area. Maybe the other messages could make the bots act differently too?

**Comment by randomperson189** [comment]
Good idea, bots do place message beacons if they have one so I'll think about it.
